src/deluge/gui/context_menu/launch_style.cpp

```diff
@@ -14,8 +14,9 @@ namespace deluge::gui::context_menu {
 enum class LaunchStyle::Value {
```

Comment:
Can this reuse the launch style enum?

Reply:
yeah it should be, that's on me. I can make that change real fast in the community trunk.